packaging/rpm/crc.spec.in

```diff
@@ -60,7 +60,7 @@ export GOFLAGS="-mod=vendor"
 mkdir embed-files
 cp /usr/bin/crc-driver-libvirt embed-files
 cp /usr/bin/crc-admin-helper embed-files/crc-admin-helper-linux
-make COMMIT_SHA=__COMMIT_SHA__ GO_EXTRA_LDFLAGS="-B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n')" GO_EXTRA_BUILDFLAGS="" CUSTOM_EMBED=true EMBED_DOWNLOAD_DIR=embed-files/ release
+make CRC_IN_DISCONNECTED_ENV=true COMMIT_SHA=__COMMIT_SHA__ GO_EXTRA_LDFLAGS="-B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n')" GO_EXTRA_BUILDFLAGS="" CUSTOM_EMBED=true EMBED_DOWNLOAD_DIR=embed-files/ release
```

Comment:
`CRC_IN_DISCONNECTED_ENV=1` is more common imo. However, I would not add this check here, as the rpm builds we are making on github for example are not disconnected. This can go in the spec file used on platforms where the builders don't have network access.